frontend/src/app/components/channel/small/small.controller.ts

```diff
@@ -189,7 +189,15 @@ export class SmallChannelController extends BaseChannelController {
         this.onlineUsers[userIndex] = lastLogin;
       }
 
+      // save current preview composite
+      var oldPrevComposite = angular.copy(this.composites[this.compositeIndex.preview]);
+
       this.constructComposites();
+
+      var newPrevComposite = angular.copy(this.composites[this.compositeIndex.preview]);
+      if(!angular.equals(oldPrevComposite, newPrevComposite)){
+        this.putCompositeToSlot(this.compositeIndex.preview, false, true);
+      }
```

Comment:
removeUserFromComposites() method only operates on the array, actual putCompositeToSlot() has to be in another place, please fix